app/src/main/java/com/example/bookwormadventuresdeluxe2/NotificationUtility/MyResponse.java

```diff
@@ -0,0 +1,9 @@
+package com.example.bookwormadventuresdeluxe2.NotificationUtility;
+
+/**
+ * Stores request result
+ */
+public class MyResponse
```

Comment:
Is this class necessary?

Reply:
I renamed this to RetrofitResponse. Pls see processNotification inside NotificationHandler for usage. The added comments should clarify.
![processNotification Inside NotificationHandler](https://user-images.githubusercontent.com/44795394/100084273-cc2bbf80-2e07-11eb-897f-5cc75492f152.PNG)

Reply:
Changed notification badge text colour to black.
Currently notification clicks take user to login page or my Books if logged in.